apps/src/makerlab/BoardController.js

```diff
@@ -0,0 +1,156 @@
+var five = require('johnny-five');
+var ChromeSerialPort = require('chrome-serialport');
+var SerialPort = ChromeSerialPort.SerialPort;
+var list = ChromeSerialPort.list;
+var isInstalled = ChromeSerialPort.isInstalled;
+var PlaygroundIO = require('playground-io');
```

Comment:
Is the problem just that you are 'dereferencing' your missing shimmed dependencies right away (lines 3, 4, 5) or does it mess something else up about the bundle?  Could try:
```javascript
var SerialPort = ChromeSerialPort && ChromeSerialPort.SerialPort;
// etc...
```
and see if that lets you require at the top of applab.js and just conditionally construct your controller.